packages/myst-parser/src/myst.ts

```diff
@@ -59,7 +60,17 @@ function parseOptions(opts?: Options): AllOptions {
 export function createTokenizer(opts?: Options) {
   const parsedOpts = parseOptions(opts);
   const { extensions, markdownit } = parsedOpts;
-  const tokenizer = MarkdownIt(MARKDOWN_IT_CONFIG as any, markdownit);
+  const tokenizer = MarkdownIt(
+    {
+      ...MARKDOWN_IT_CONFIG,
+      options: {
+        ...MARKDOWN_IT_CONFIG.options,
+        typographer: extensions.smartquotes ? true : false,
```

Comment:
Cool, so this is required to enable any of the typographic features - if it's `false` smartquotes don't get replaced because of this: https://github.com/markdown-it/markdown-it/blob/master/lib/rules_core/smartquotes.js#L190

Reply:
I beleive that turning on typographer is probably a performance optimization in the library. So only enabling it here when we need it seemed like a good idea?